crates/client-api/src/auth.rs

```diff
@@ -70,9 +84,18 @@ impl SpacetimeCreds {
 #[derive(Clone)]
 pub struct SpacetimeAuth {
     pub creds: SpacetimeCreds,
-    pub identity: Identity,
-    pub subject: String,
-    pub issuer: String,
+    pub claims: SpacetimeIdentityClaims,
+    // The decoded JWT payload.
```

Comment:
```suggestion
    /// The decoded JWT payload.
```

Use `///` for doc comments.